docs/user/reference/environment-variables.rst

```diff
@@ -51,6 +51,19 @@ All :doc:`build processes </builds>` have the following environment variables au
 
     :Example: ``/home/docs/checkouts/readthedocs.org/user_builds/project/envs/version``
 
+.. envvar:: READTHEDOCS_OUTPUT
+
+    Base path for output formats when using custom :doc:`build.commands </build-customization>` to generate output.
```

Comment:
This variable always works, not only when using build.commands. I don't think we need to mention any type of build customization here.

Reply:
```suggestion
    Base path for well-known output format directories. Files in these directories will automatically be found, uploaded and published.

```

Reply:
This looks better!